extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -250,10 +251,50 @@ public DateTime getStartTime(final String id)
       return null;
     }
     catch (IOException e) {
-      throw Throwables.propagate(e);
+      throw new RuntimeException(e);
     }
   }
 
+  public Map<String, Object> getMovingAverages(final String id)
+  {
+    log.debug("GetMovingAverages task[%s]", id);
+
+    try {
+      final FullResponseHolder response = submitRequest(
+          id,
+          HttpMethod.GET,
+          "rowStats",
+          null,
+          true
+      );
+      return response.getContent() == null || response.getContent().isEmpty()
+             ? ImmutableMap.of()
+             : jsonMapper.readValue(response.getContent(), new TypeReference<Map<String, Object>>()
```

Comment:
`JacksonUtils` has a field for this type reference

Reply:
cool, used the type reference from JacksonUtils